share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -666,6 +666,85 @@ ml-darwin-aarch64-mps-build:
     - artifacts: True
       job: ml-darwin-aarch64-mps-generate
 
+##########################################
+# Machine Learning - Linux power64le (CPU)
+##########################################
+.ml-linux-power64le-cpu:
+  extends: [ ".linux_power" ]
+  variables:
+    SPACK_CI_STACK_NAME: ml-linux-power64le-cpu
+
+.tags-ml-power:
+  tags: ["spack", "public", "large", "ppc64le"]
+
+ml-linux-power64le-cpu-ubuntu-generate:
+  extends: [ ".generate", ".ml-linux-power64le-cpu", ".tags-ml-power" ]
+  image: { "name": "ecpe4s/ubuntu20.04-runner-ppc64le:2023-01-01", "entrypoint": [""] }
+
+ml-linux-x86_64-cpu-ubuntu-build:
+  extends: [ ".build", ".ml-linux-power64le-cpu" ]
+  trigger:
+    include:
+      - artifact: jobs_scratch_dir/cloud-ci-pipeline.yml
+        job: ml-linux-power64le-cpu-ubuntu-generate
+    strategy: depend
+  needs:
+    - artifacts: True
+      job: ml-linux-power64le-cpu-ubuntu-generate
+
+ml-linux-power64le-cpu-fedora-generate:
+  extends: [ ".generate", ".ml-linux-power64le-cpu", ".tags-ml-power" ]
+  image: { "name": "ecpe4s/fedora36-runner-ppc64le:latest", "entrypoint": [""] }
```

Comment:
This image does not seem to exist. At least that's what it says [here](https://gitlab.spack.io/spack/spack/-/jobs/7843299#L10), and I can't seem to find it on dockerhub.

It seems to be mentioned elsewhere in the PR too, so I'll just point it out here.

Reply:
Yes, I have not been able to find an official list of what containers are available.  Todd pointed me at https://github.com/spack/gitlab-runners , but that doesn't specify the final image names and tags available, so I'm having to guess and check.  That page does seem to imply that there is a fedora36 image that supports ppc64le, but unclear what the exact name and tag are.  No other pipelines currently appear to use it either.

Reply:
We maintain the `ecpe4s` repo on DockerHub and we've never made a fedora36 image by the name.

What we have at `ecpe4s` are the following images:

AMD64
* `ecpe4s/ubuntu20.04-runner-amd64-gcc-11.4:2023.08.01`
* `ecpe4s/ubuntu20.04-runner-amd64-gcc-12.3:2023.08.01`

PPC64LE
* `ecpe4s/ubuntu20.04-runner-ppc64le-gcc-11.4:2023.08.01`
* `ecpe4s/ubuntu20.04-runner-ppc64le-gcc-12.3:2023.08.01`

ARM64
* `ecpe4s/ubuntu20.04-runner-arm64-gcc-11.4:2023.08.01`
* `ecpe4s/ubuntu20.04-runner-arm64-gcc-12.3:2023.08.01`

Please see this repo for the recipes:
* https://github.com/UO-OACISS/e4s/tree/master/docker-recipes/runner

Reply:
What about `ecpe4s/rhel8-runner-ppc64le:2023-01-01`?  That would seem to satisfy my need for a RHEL-based runner on power architecture.  It hasn't been updated since the new year, but I've worked with worse.  Also unclear if the recent RHEL licensing changes will affect the long-term usability of such an image.